### PR TITLE
SemVer scripts added

### PR DESCRIPTION
**Pull request by Andkoo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "mangata-bridge",
+  "version": "0.1.0",
+  "repository": "https://github.com/mangata-finance/mangata-bridge.git",
+  "author": "Mangata Finance",
```
Comment (gleb-urvanov) [inline package.json:5]: Snowfork is original autor, we only introduced minor changes here.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "license": "MIT",
```
Comment (gleb-urvanov) [inline package.json:6]: Update the license according to actual one used in the repo.

**Review by gleb-urvanov — CHANGES_REQUESTED** [review]

**Review by iStrike7 — APPROVED** [review]

**Review by iStrike7 — CHANGES_REQUESTED** [review]
There's no license anymore?

**Comment by Andkoo** [comment]
> There's no license anymore?

Fixed

**Review by iStrike7 — APPROVED** [review]

**Review by gleb-urvanov — APPROVED** [review]

**Pull request merged**